### DISCUSSION TITLE
Force-restart chrome extensions

### DISCUSSION THREAD

**Discussion by 3kh0** [3kh0/ext-remover#359] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
This kajig can force many chrome extensions to restart. It can open a small window of time to open a blocked website, and has proven useful to me in the past.

Step 1: Go to chrome://extensions/. If this is blocked, you can't do this.
Step 2: Find the extension you want to restart.
Step 3: Click the Details button.
Step 4: Look for the Allow acces to file URLs toggle. If this toggle is absent, try toggles with similar names.
Step 5: Allow access, and then quickly revoke access. The extension should restart.

Repeat steps 4-5 until you want to extension to load fully.

**Comment by Aka-but-nice** [comment]
I really can't test this because ever since I pressed hard kill, even after I restart, signout, or shutdown, securly is gone.
